local-rest-scorer/src/test/java/ai/h2o/mojos/deploy/local/rest/controller/ModelsApiControllerTest.java

```diff
@@ -0,0 +1,174 @@
+package ai.h2o.mojos.deploy.local.rest.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ai.h2o.mojos.deploy.common.rest.model.CapabilityType;
+import ai.h2o.mojos.deploy.common.transform.MojoScorer;
+import ai.h2o.mojos.deploy.common.transform.SampleRequestBuilder;
+import ai.h2o.mojos.deploy.common.transform.ShapleyLoadOption;
+import ai.h2o.mojos.runtime.MojoPipeline;
+import ai.h2o.mojos.runtime.api.BasePipelineListener;
+import ai.h2o.mojos.runtime.api.MojoPipelineService;
+import ai.h2o.mojos.runtime.frame.MojoColumn;
+import ai.h2o.mojos.runtime.frame.MojoFrame;
+import ai.h2o.mojos.runtime.frame.MojoFrameBuilder;
+import ai.h2o.mojos.runtime.frame.MojoFrameMeta;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+
+@ExtendWith(MockitoExtension.class)
+class ModelsApiControllerTest {
+  private static final String MOJO_PIPELINE_PATH = "src/test/resources/multinomial-pipeline.mojo";
+  private static final String TEST_UUID = "TEST_UUID";
+
+  @Mock private SampleRequestBuilder sampleRequestBuilder;
+
+  @BeforeAll
+  static void setup() {
+    System.setProperty("mojo.path", "src/test/resources/multinomial-pipeline.mojo");
+    mockDummyPipeline();
+  }
+
+  private static void mockDummyPipeline() {
+    MojoPipeline dummyPipeline =
+        new DummyPipeline(TEST_UUID, MojoFrameMeta.getEmpty(), MojoFrameMeta.getEmpty());
```

Comment:
I wonder if our static mock troubles would be fixed if we moved the `loadMojoPipelineFromFile()` call in `MojoScorer.java` from static to the constructor.

Reply:
maybe... Seems like a bigger rework than I'm inclined to tackle at the moment.